pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -130,7 +129,9 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, r *http.Requ
 	doc.OpenShiftCluster.ID, doc.OpenShiftCluster.Name, doc.OpenShiftCluster.Type = oldID, oldName, oldType
 
 	if isCreate {
+		doc.ClusterResourceGroupIDKey = strings.ToLower(doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID)
 		doc.OpenShiftCluster.Properties.ProvisioningState = api.ProvisioningStateCreating
+		doc.OpenShiftCluster.Properties.ClusterProfile.Version = "4.3.0-0.nightly-2019-12-05-001549"
```

Comment:
?

Reply:
this is where we initialise the cluster version that is going to be deployed.  Until it is a nice simple string like "4.3.0" that the user can choose, I'm hard-coding it